docs/diff-strategy.md

```diff
@@ -30,6 +30,12 @@ However, this is a fairly new feature and only available for Kubernetes versions
 greater than 1.13. Only the API server (master nodes) needs to have that version, worker nodes
 do not matter.
 
+There is also a [longstanding bug in
+Kubernetes](https://github.com/kubernetes/kubernetes/issues/39188) that results
+in wrong diff results if the same port number gets multiple protocols assigned,
+most commonly if a port is specified as a UDP and a TCP port. `tk apply` will
+still do the right thing in that case.
```

Comment:
I feel like this reads a bit easier:

```markdown
There is also a [long-standing bug in
`kubectl`](https://github.com/kubernetes/kubernetes/issues/39188) that results
in an incorrect diff output if the same port number is defined twice,
which commonly happens if a port is specified using both, `tcp` and `udp`.
Nevertheless, `tk apply` will still work correctly.
```

Reply:
Yeah, that would make more sense if that bug only affected the TCP/UDP use case. Perhaps that's actually the case. I'm fine with assuming that until proven otherwise.